core/block.js

```diff
@@ -44,6 +44,7 @@ const constants = goog.require('Blockly.constants');
 const fieldRegistry = goog.require('Blockly.fieldRegistry');
 const idGenerator = goog.require('Blockly.utils.idGenerator');
 const inputTypes = goog.require('Blockly.inputTypes');
+const messages = goog.require('Blockly.utils.messages');
```

Comment:
@cpcallen Since this is new, does this need an `exports.messages` assignment? or no?

Reply:
If it is intended to be part of the Blockly API then yes: it should be reexported in `utils.js`.  (Same for `browserEvents` added in #5464.)

Reply:
Cool cool 👍 I think browser events are fine though because it's just `Blockly.browserEvents`, and it already existed.

Reply:
You are quite correct.  Evidently it is getting late here…

Reply:
node did not like it and I don't know why :/ Last commit passes tests just fine but after adding `exports.messages = messages` it gets angry.